migrations/20200719025533-add-name-column-users.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction(async (t) => {
+      await queryInterface.addColumn(
+        "users",
+        "name",
+        {
+          type: Sequelize.DataTypes.STRING,
+        },
+        { transaction: t }
+      )
+      await queryInterface.sequelize.query(
+        `UPDATE users SET "name" = CONCAT("firstName", ' ', "lastName")`,
+        { transaction: t }
+      )
+      await queryInterface.removeColumn('users', 'firstName', { transaction: t });
+      return queryInterface.removeColumn('users', 'lastName', { transaction: t });
+    });
+  },
+
+  down: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction(async (t) => {
+      await queryInterface.addColumn(
+        "users",
+        "firstName",
+        {
+          type: Sequelize.DataTypes.STRING,
+        },
+        { transaction: t }
+      );
+      await queryInterface.addColumn(
+        "users",
+        "lastName",
+        {
+          type: Sequelize.DataTypes.STRING,
+        },
+        { transaction: t }
+      );
+      await queryInterface.sequelize.query(
+        'UPDATE users SET "firstName" = "name", "lastName" = "name"',
```

Comment:
this isn't super sustainable - the names get too long if you run undo a bunch of times (like i did during testing); if you can find an easy way to do a slightly different operation (e.g. explode `name` on ' ' and put the first part in `firstName` and second part in `lastName`) that would be preferred, otherwise just lemme know and we'll flag it to be fixed later

Reply:
would it be better to do a similar thing like you did with admin and ``UPDATE users SET "firstName" = "name", "lastName" = ''``

Reply:
let's just leave it as is, but maybe add a comment here to say running this down migration too often may result in issues